Source/misdat.cpp

```diff
@@ -99,8 +99,8 @@ MissileData missiledata[] = {
 	{  &AddManaTrap,               &MI_Dummy,          MIS_MANATRAP,      false,     1, MISR_NONE,      MFILE_NONE,      IS_CAST7,    SFX_NONE    },
 	{  &AddRing,                   &MI_LightningRing,  MIS_LIGHTRING,     false,     1, MISR_LIGHTNING, MFILE_LGHNING,   SFX_NONE,    SFX_NONE    },
 	{  &AddSearch,                 &MI_Search,         MIS_SEARCH,        false,     1, MISR_NONE,      MFILE_NONE,      SFX_NONE,    SFX_NONE    },
-	{  &AddFlashBack,              &MI_FlashBack,      MIS_FLASHBK,       true,      1, MISR_MAGIC,     MFILE_BLUEXBK,   SFX_NONE,    SFX_NONE    },
-	{  &AddFlashFront,             &MI_FlashFront,     MIS_FLASHFR,       true,      1, MISR_MAGIC,     MFILE_BLUEXFR,   SFX_NONE,    LS_ELECIMP1 },
+	{  nullptr,                    nullptr,            MIS_FLASHBK,       false,     1, MISR_MAGIC,     MFILE_BLUEXBK,   SFX_NONE,    SFX_NONE    },
+	{  nullptr,                    nullptr,            MIS_FLASHFR,       false,     1, MISR_MAGIC,     MFILE_BLUEXFR,   SFX_NONE,    LS_ELECIMP1 },

```

Comment:
I'm not sure what's going on here?

Reply:
deleting 4 unused functions but keeping the spell indexes aligned.